meta/Makefile

```diff
@@ -41,7 +41,6 @@ WARNINGS = \
 	-Winit-self \
 	-Winline \
 	-Winvalid-pch \
-	-Wlong-long \
```

Comment:
why this is removed ?

Reply:
Need  ULL macro suffix for unsigned long long incase of 32 bit arch, for a 64bit const